src/main/java/careconnect/logic/commands/AddLogCommand.java

```diff
@@ -109,7 +111,7 @@ public boolean equals(Object other) {
     @Override
     public String toString() {
         return new ToStringBuilder(this)
-                .add("log", log)
+                .add("toAddLog", log)
```

Comment:
I am sorry but correct me if i m wrong: based on other commands, the string should match the field name. So we should either rename the field `log` to `toAddLog` or change the string back to just `log`.